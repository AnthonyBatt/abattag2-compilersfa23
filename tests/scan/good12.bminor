"I passed strings to my encode stuff properly\n"
"\t that was a tab\t so was that\n"
"this is 6 this is also 6 but in hex \0x36\n"
