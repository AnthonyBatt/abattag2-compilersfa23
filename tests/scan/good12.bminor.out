TOKEN_STRING_LITERAL      I passed strings to my encode stuff properly

TOKEN_STRING_LITERAL      	 that was a tab	 so was that

TOKEN_STRING_LITERAL      this is 6 this is also 6 but in hex 6

